wasm-chain-integration/src/v1/trie/types.rs

```diff
@@ -227,6 +228,11 @@ pub trait Loadable: Sized {
     }
 }
 
+/// A trait implemented by types that can be displayed as a tree.
```

Comment:
It is not really that general since it has the BackingStoreLoad requirement.

Since this is so specific to displaying our tree I would prefer not to have this trait, the same can be achieved by adding methods to `Node` and `PersistentState`.

Reply:
agree